tests/conftest.py

```diff
@@ -2705,11 +2705,29 @@ def flow_cell_directory_name_demultiplexed_with_bcl_convert(
     return f"230504_A00689_0804_B{flow_cell_name_demultiplexed_with_bcl_convert}"
 
 
-@pytest.fixture
-def demultiplexed_flow_cells_directory(tmp_path) -> Path:
+@pytest.fixture(name="demultiplexed_flow_cells_directory")
```

Comment:
I would like to distinguish between directories created using pytest's tmp_path/tmp_dir and directories in `tests/fixtures`. Could you include this in the name? Maybe `demultiplexed_flow_cells_tmp_directory`.

Reply:
Yes, I agree. I changed the name